.ci/appveyor.yml

```diff
@@ -56,6 +56,15 @@ install:
   - "npm config set loglevel warn"
   - "npm install"
 
+  - curl -fsSL ftp://ftp.zlatkovic.com/libxml/iconv-1.9.2.win32.zip -o iconv.zip
```

Comment:
Appveyor has lots of non-standard Windows software installed, especially msys & cygwin. See https://www.appveyor.com/docs/installed-software/